src/main/java/com/buuz135/industrial/config/CustomConfiguration.java

```diff
@@ -1,24 +1,27 @@
 package com.buuz135.industrial.config;
 
+import java.util.HashMap;
+
 import com.buuz135.industrial.tile.block.CustomOrientedBlock;
-import net.minecraftforge.common.config.Configuration;
 
-import java.util.HashMap;
+import net.minecraftforge.common.config.Configuration;
 
 public class CustomConfiguration {
 
-    public static Configuration config;
-    public static HashMap<String, Boolean> configValues;
-    public static boolean enableBookEntriesInJEI;
+	public static Configuration config;
+	public static HashMap<String, Boolean> configValues;
+	public static boolean enableBookEntriesInJEI;
+	public static boolean enableAdditionalLens;
 
-    public static void sync() {
-        try {
-            CustomOrientedBlock.blockList.forEach(CustomOrientedBlock::getMachineConfig);
-            enableBookEntriesInJEI = config.getBoolean("enableBookEntriesInJEI", Configuration.CATEGORY_CLIENT, true, "Enable to show book entries in JEI");
-        } catch (Exception e) {
+	public static void sync() {
```

Comment:
The formatting on this entire PR is off, you should be using the same formatting as IF normally does

Reply:
I was just using Eclipse default, is there a way to format it according to your standard?